src/commonComponents/ConfirmPopup.tsx

```diff
@@ -0,0 +1,88 @@
+import { Button, Modal } from 'flowbite-react';
+
+interface IProps {
+    openModal: boolean;
+    closeModal: (flag: boolean) => void;
+    onSuccess: (flag: boolean) => void;
+    message: string
+    isProcessing: boolean
+}
+
+const ConfirmModal = (props: IProps) => {
```

Comment:
Can you please be specific for the model name

Reply:
This is a custom confirmation popup and we can use it anywhere in our code.
Can you please suggest a better name?